controllers/bootstrap/init.go

```diff
@@ -1374,11 +1376,48 @@ func (b *Bootstrap) updateApprovalMode() error {
 		return err
 	}
 
-	if err = b.UpdateCsOpApproval(); err != nil {
+	if err = b.UpdateOpApproval("ibm-common-service-operator"); err != nil {
 		klog.Errorf("Failed to update common service operator subscription: %v", err)
 		return err
 	}
 
+	if err = b.UpdateOpApproval("operand-deployment-lifecycle-manager-app"); err != nil {
+		klog.Errorf("Failed to update operand-deployment-lifecycle-manager-app subscription: %v", err)
+		return err
+	}
+
+	if err = b.UpdateOpApproval("ibm-namespace-scope-operator"); err != nil {
```

Comment:
I am wondering if we are able to use constant variable instead of hard-coded string for Operator Name.

We could find `ibm-common-service-operator` here
https://github.com/IBM/ibm-common-service-operator/blob/master/controllers/constant/constant.go#L68

Reply:
sorry, I just saw it. I will fix them now, thanks a lot for advice! 😄